model-server/src/main/kotlin/org/modelix/model/server/Main.kt

```diff
@@ -159,6 +161,8 @@ object Main {
                 install(Routing)
                 installAuthentication(unitTestMode = !KeycloakUtils.isEnabled())
                 install(ForwardedHeaders)
+                install(Resources)
+                install(IgnoreTrailingSlash)
```

Comment:
Maybe link this as the reason for installing the plugin: https://opensource.zalando.com/restful-api-guidelines/#136

Reply:
done